tools/system_libs.py

```diff
@@ -1776,6 +1784,8 @@ def add_library(libname):
     add_library('libc++abi')
     if settings.EXCEPTION_HANDLING:
       add_library('libunwind')
+  if settings.FORMAT_EXCEPTION_SUPPORT:
```

Comment:
Ideally I think we could do add this without adding a new setting or a new library

Reply:
I would appreciate your advice on the best approach for exposing this.